test/e2eTests/tests/dashboard_test.js

```diff
@@ -1,12 +1,54 @@
 
 Feature('Dashboard');
 
+const LOG_TAG = '[DashboardTests]';
+
 // In order to be on the dashboard we must first login
-Before((I) => {
+Before(async (I) => {
   I.login();
 });
 
 Scenario('Logout successfully', (I, DashboardPage) => {
-	const logoutStatus = DashboardPage.logout();
-	logoutStatus.successful();
+  const logoutStatus = DashboardPage.logout();
+  logoutStatus.successful();
+});
+
+Scenario('Navigate though dashboards and transfer the patient between dashboards', async (I, Apis, Data, DashboardPage, RegistrationDashboardPage) => {
+  I.say(`${LOG_TAG} Creating the patient`);
+  Patient = await Apis.patient.create(Data.patients.patient2);
+
+  I.say(`${LOG_TAG} Navigate to the registration page`);
+  const registrationPage = DashboardPage.navigateToRegistrationPage();
+
+  I.say(`${LOG_TAG} Disable auto-select`);
+  registrationPage.disableAutoSelect();
+
+  I.say(`${LOG_TAG} Search for patient's identifier`);
+  registrationPage.searchForPatientByIdAndSelect(Data.patients.patient2);
+
+  I.say(`${LOG_TAG} Make sure the registration dashboard page is loaded`);
+  RegistrationDashboardPage.isLoaded();
+
+  RegistrationDashboardPage.clickTransferPatientButton();
+
+  const socialDashboardPage = RegistrationDashboardPage.transferToSocialModule();
+  socialDashboardPage.clickTransferPatientButton();
```

Comment:
Would be nice to have logs here.

Reply:
The methods log some messages like "Clicking on the transfer button" and "Transferring the patient to social module"